gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/CoopLockFsckRunner.java

```diff
@@ -114,6 +114,7 @@ public int run() throws IOException {
             .map(this::getOperationLockIfExpiredUnchecked)
             .filter(Optional::isPresent)
             .map(Optional::get)
+            .filter(operation -> !unlockOperationIfNoLogUnchecked(operation))
```

Comment:
Can we make this not double negative way?

Reply:
Done.